components/public-api-server/pkg/oidc/service.go

```diff
@@ -137,6 +137,16 @@ func randString(size int) (string, error) {
 
 func (s *Service) GetClientConfigFromStartRequest(r *http.Request) (*ClientConfig, error) {
 	orgSlug := r.URL.Query().Get("orgSlug")
+	idParam := r.URL.Query().Get("id")
+
+	// if no org slug is given, we assume the request is for the default org
+	if orgSlug == "" && idParam == "" {
+		org, err := db.GetSingleTeam(r.Context(), s.dbConn)
+		if err != nil {
+			return nil, fmt.Errorf("Failed to find team: %w", err)
+		}
+		orgSlug = org.Slug
+	}
```

Comment:
Is it problematic that I can craft this request manually and submit it with/without the slug when there may already be existing setup completed?

Reply:
I don't think so. It would do the same login process in both cases.